src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -4756,7 +4757,7 @@ def test_derived_type_arg():
         _, invoke_info = parse(
             os.path.join(BASE_PATH,
                          "1.6.2_single_invoke_1_int_from_derived_type.f90"),
-            api=TEST_API, distributed_memory=dist_mem)
+            api=TEST_API)
```

Comment:
Please update the tests that you've touched to use the `dist_mem` fixture.

Reply:
Core blimey that was a lot of changes.